src/lib/stores/root.ts

```diff
@@ -22,7 +22,7 @@ export class RootStore {
 
   localChainConfigStore: LocalChainConfigStore;
 
-  verifyModuleTaskStore: VerifyModuleTaskStore;
+  MoveVerifyTaskStore: MoveVerifyTaskStore;
```

Comment:
Should we use lowercase for the first letter? To make the naming convention more according to others